data/net/minecraft/core/Cursor3D.mapping

```diff
@@ -1 +1,15 @@
-CLASS net/minecraft/core/Cursor3D
+CLASS net/minecraft/core/Cursor3D net/minecraft/core/Cursor3D
+	COMMENT This class represents a cuboid. You can iterate over it using the methods {@link Cursor3D#advance()} and {@link Cursor3D#nextX()}, {@link Cursor3D#nextY()}, {@link Cursor3D#nextZ()}
+	METHOD <init> (IIIIII)V
+		ARG 1 originX
+		ARG 2 originY
+		ARG 3 originZ
+		ARG 4 x2
+		ARG 5 y2
+		ARG 6 z2
```

Comment:
I was mapping this class at some point before I had to go do other things, I think the best names for these would be 
```
destinationX,
destinationY
destinationZ
```


Simply to match with the `originX` , `originY` and `originZ` that is already there.